src/ResourceManager/ServiceBus/Commands.ServiceBus/Cmdlets/Queue/NewAzureServiceBusQueue.cs

```diff
@@ -193,17 +193,7 @@ public override void ExecuteCmdlet()
                 queueAttributes.RequiresSession = RequiresSession;
 
             if (SizeInBytes != null)
-                queueAttributes.SizeInBytes = SizeInBytes;
-
-#pragma warning disable 612, 618
-            queueAttributes.Location = getNamespaceLoc.Location;
-
-            if (EnableBatchedOperations != null)
```

Comment:
Do these parameters serve any purpose now that you have removed this code? If not, please remove the parameters.

Reply:
removed the parameters